src/Cache/ContaoCacheWarmer.php

```diff
@@ -116,8 +116,17 @@ private function generateConfigCache($cacheDir)
     {
         $dumper = new CombinedFileDumper($this->filesystem, new PhpFileLoader(), $cacheDir.'/contao', true);
 
-        $dumper->dump($this->locator->locate('config/autoload.php', null, false), 'config/autoload.php');
-        $dumper->dump($this->locator->locate('config/config.php', null, false), 'config/config.php');
+        try {
+            $dumper->dump($this->locator->locate('config/autoload.php', null, false), 'config/autoload.php');
+        } catch (\InvalidArgumentException $e) {
+            // No autoload.php files found
+        }
```

Comment:
IMO this should be handled by a method:
```php
private function findConfig($fileName)
{
    try {
        return $this->locator->locate('config/' . $fileName, null, false);
    } catch (\InvalidArgumentException $e) {
        return [];
    }
}
```
This not only lowers the complexity of this method but also will not mask away any `\InvalidArgumentException` which could be thrown within the `$dumper->dump()` call but will only catch the ones originating from the `FileLocator`.

Same below for the `autoload.php`